uniffi_bindgen/src/interface/mod.rs

```diff
@@ -241,6 +241,22 @@ impl<'ci> ComponentInterface {
         }
     }
 
+    pub fn contains_optional_types(&self) -> bool {
```

Comment:
I really don't like the redundancy here and wish I could pass the sought type into a generic `countains _type` function but I have learned that "destructuring assignments are unstable" and explicitly covering all of the enum variants for these types would be silly. I'm not sure if this is macro territory but if you have any suggestion for how to refactor this I am open.

Reply:
Yeah, I see what you mean. I don't think a macro would help maintainability here so I'm happy to just eat the redundancy.